### DISCUSSION TITLE
typecasting

### DISCUSSION THREAD

**Discussion by frepog** [community/community#27899] [Other Feature Feedback, Questions, & Ideas]
anyone know what is typecasting https://i.imgur.com/ZbcsFTG.png like this because i don't know where to find it and where to learn about it

**Comment by jfullstackdev** [comment]
I don't know what language is that but here is one example from Java 

https://www.w3schools.com/java/java_type_casting.asp

**Reply by frepog** [reply]
c++

**Comment by EliasAfara** [comment]
> anyone know what is typecasting https://i.imgur.com/ZbcsFTG.png like this because i don't know where to find it and where to learn about it

Typecasting means the conversion of a variable type to another.

I recommend you to watch [this](https://www.youtube.com/watch?v=uI10H1jzw00&ab_channel=NesoAcademy) youtube video for some explanation and check out [this](https://www.javatpoint.com/type-casting-in-cpp) and [this](https://www.educba.com/type-casting-in-c-plus-plus/) for code examples.

**Reply by frepog** [reply]
<3

**Reply by EliasAfara** [reply]
@frepog  mark it as an answer to close the discussion.

**Comment by jfullstackdev** [comment]
here for C++

https://www.geeksforgeeks.org/type-conversion-in-c/

if you still have questions, you can post it here
